java-chassis-reference/en_US/general-development/local-develop-test.md

```diff
@@ -96,28 +99,32 @@ localserv:
         - rest://localhost:8080
         - highway://localhost:7070
 ```
-* **步骤 2**consumer本地部署契约文件
+* **Step 2**consumer local deployment contract file
 
-参考：[定义服务契约](https://huaweicse.github.io/servicecomb-java-chassis-doc/zh_CN/build-provider/define-contract.html)
-* **步骤 3**在consumer main函数，启动ServiceComb引擎之前声明：
+Reference: [Define Service Contract] (https://docs.servicecomb.io/java-chassis/zh_CN/build-provider/define-contract.html)
+* **Step 3** In the consumer main function, declare the ServiceComb engine before starting:
 
 ```java
 　　System.setProperty("local.registry.file", "/path/registry.yaml");
 ```
 
-setProperty第二个参数填写registry.yaml在磁盘中的系统绝对路径，注意区分在不同系统下使用对应的路径分隔符。
+The second parameter of setProperty fills in the absolute path of the registry.yaml system on the disk, pay attention to distinguish the corresponding path separator in different systems.
+
 
+## Local debugging by setting environment information
+The java chassis is strictly dependent on the contract when designing, so usually the version of the microservice has to change when the contract updated. However, if the development mode is still in progress, it is normal to modify the interface.   
+If you need to change the version every time, it is very unfriendly to the user, so an environment setting is added for this case. If the microservice is configured as a development environment, the interface is modified (the schema has changed), and the restart can be registered to the service center without modifying the version number.  
```

Comment:
If you need to change the version every time ->
If need to change the version every time